src/Tolerance/Bridge/Laravel/Resources/config/tolerance.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+    'message_profile' => [
```

Comment:
Ah, this bit is for tolerance configuration. Unlike in Symfony, you don't have anything to define a configuration tree with validation, so not really sure how to do it to be honest. The practice seems to just tell the user what the config can be and let him bear all responsibilities if he messes up.